packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -391,8 +390,11 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
     return focusZoneProps && focusZoneProps.direction !== undefined ? focusZoneProps.direction : FocusZoneDirection.vertical;
   }
 
-  private _onRenderSubMenu(subMenuProps: IContextualMenuProps) {
-    return <ContextualMenu {...subMenuProps} />;
+  private _onRenderSubMenu(subMenuProps: IContextualMenuProps, defaultRenderer: any) {
```

Comment:
Shouldn't need to specify `defaultRenderer` arg if it's not used.

Reply:
Actually, if this whole thing isn't supposed to be used, can't we remove it in it's entirety?

Reply:
This parameter was not there in the first iteration. I saw a tslint giving me the error on [L350](https://github.com/OfficeDev/office-ui-fabric-react/blob/55d5a01747ff47115e3a20fdf367209693838385/packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx#L350) about `this._onRenderSubMenu` being not compatible as argument. This is a little bit wierd and I will confirm this tomorrow.

Reply:
@joschect The primary issue is if someone reuses `ContextualMenu.base` and doesn't provide the `onRenderSubMenu` prop. This warning will tell them one needs to be provided.

Reply:
Actually the `_onRenderMenuList` defined at [~L400](https://github.com/OfficeDev/office-ui-fabric-react/blob/3974eef14189b377c87d60fa7b2cf155a180da58/packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx#L398) below has a similar unused parameter `defaultRender`. So I think it needs to exist for sake the type inference for [~L229-L230](https://github.com/OfficeDev/office-ui-fabric-react/blob/3974eef14189b377c87d60fa7b2cf155a180da58/packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx#L229-L230). Then it's weird tslint didn't warn me about type incompatibility of `onRenderSubMenu` in the first iteration.